docs/languages/csharp/distinguish-delegates-events.md

```diff
@@ -1,8 +1,80 @@
 # Distinguising Delegates and Events
 
+By [Bill Wagner](https://github.com/BillWagner)
+
 [Previous](modern-events.md)
 
-This article will provide guidance on how to
-decide between using delegates and events in your design.
+Developers that are new to the .NET Core platform often struggle
+when deciding between a design based on `delegates` and a design
+based on `events`. This is a difficult concept, because the two
+language features are very similar. Events are even built using
+the language support for delegates. 
+
+They both offer a late binding scenario: they enable scenarios
+where a component communicates by calling a method that is only
+known at runtime. They both support single and multiple subscriber
+methods. You may find this referred to as singlecast and multicast
+support. They both support similar syntax for adding and removing
+handlers. Finally, raising an event and calling a delegate use exactly the same method call syntax. They even both support the same `Invoke()`
+method syntax for use with the `?.` operator.
+
+With all those similarities, it is easy to have trouble determining when
+to use which.
+
+# Listening to Events is Optional
+
+The most important consideration in determining which language feature
+to use is whether or not there must be an attached subscriber. If your
+code can't complete its tasks without an attached subscriber, you should
+use a design based on delegates. If the absence of any subscribers
+does not affect your code's ability to complete, you should use a
+design based on events. 
+
+Consider the examples built during this section. The code you built
```

Comment:
The double negatives in this paragraph may also be worth flipping to a use of positives.